sunpy/visualization/imageanimator.py

```diff
@@ -936,16 +936,16 @@ class ImageAnimatorWCS(ImageAnimator):
     Extra keywords are passed to imshow.
 
     """
-    def __init__(self, data, wcs=None, image_axes=[-1, -2], unit_x_axis=None, unit_y_axis=None,
-                 axis_ranges=None, **kwargs):
+    def __init__(self, data, wcs=None, image_axes=[-2, -1], unit_x_axis=None, unit_y_axis=None,
```

Comment:
Then this doesn't need to be a kwarg.  You can calculate ```slice_axes``` from ```image_axes``` by assuming the wcs axes are reversed.  Right?